docs/package.json

```diff
@@ -29,7 +29,7 @@
   },
   "repository": {
     "type": "git",
-    "url": "https://github.com/strongloop/loopback-next",
+    "url": "https://github.com/strongloop/loopback-next.git",
```

Comment:
My personal preference is `"url": "https://github.com/strongloop/loopback-next" to avoid redirect.

Reply:
Fixed